### ISSUE TITLE
Clarification: 5-stall intermediate section

### ISSUE THREAD

**Issue by NoMercyO** [Zamiell/hanabi-conventions#224]
https://github.com/Zamiell/hanabi-conventions/blob/master/Reference.md#5-stalls-intermediate-section

Clarity required on what constitutes a 'playable 5' 
- Play clue through finesse?
- Play clue through prompt?
- Play clue through that is a prompt technically, but good touch principle makes obvious?
- Delay play clue through other explcitiely clued cards (all other cards are already play clued)?
- 4 is on the stack - immediately playable?

**Comment by Zamiell** [comment]
> Play clue through finesse?
no
> Play clue through prompt?
no
> Play clue through that is a prompt technically, but good touch principle makes obvious?
what
> 4 is on the stack - immediately playable?
yes

**Comment by NoMercyO** [comment]
What about a delayed play clue ( i assume yes)

**Comment by Zamiell** [comment]
yes

**Comment by Zamiell** [comment]
are you going to submit a PR or something?

**Comment by NoMercyO** [comment]
I assumed this was on hold until #226 was resolved. 

The phrase in contention:
"5 Stalls cannot be given if there are any playable 5's. In this situation, a number 5 clue would just look like a Play Clue on the 5." - i think this is just straight up wrong, or should be.

**Comment by Zamiell** [comment]
ok, marking as blocked by #226

**Comment by Zamiell** [comment]
marking as blocked by #260

**Comment by Zamiell** [comment]
#260 is accepted now.

mercy should i close this issue?
